tests/transforms/test_split_qscript.py

```diff
@@ -0,0 +1,346 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pytest
+
+import pennylane as qml
+from pennylane import numpy as pnp
+from pennylane.queuing import AnnotatedQueue
+from pennylane.tape import QuantumScript
+from pennylane.transforms import split_qscript
+
+dev = qml.device("default.qubit", wires=4)
+"""Defines the device used for all tests"""
+
+H1 = qml.Hamiltonian([1.5, 1.5], [qml.PauliZ(0) @ qml.PauliZ(1), qml.PauliZ(0) @ qml.PauliZ(1)])
```

Comment:
Nice test suite! You could remove a lot of redundant code :+1:  Could you add this test suite to the new return type folder (transforms), therefore we can check that nothing is failing?